src/main/java/seedu/trackascholar/model/applicant/Applicant.java

```diff
@@ -63,22 +63,42 @@ public Name getName() {
         return name;
     }
 
+    public String getFullName() {
+        return name.getFullName();
+    }
+
     public Phone getPhone() {
         return phone;
     }
 
+    public String getPhoneNumber() {
+        return phone.getNumber();
+    }
+
     public Email getEmail() {
         return email;
     }
 
+    public String getEmailAddress() {
+        return email.getEmailAddress();
+    }
+
     public Scholarship getScholarship() {
         return scholarship;
     }
 
+    public String getScholarshipName() {
+        return scholarship.getScholarship();
+    }
+
     public ApplicationStatus getApplicationStatus() {
         return applicationStatus;
     }
 
+    public String getStatusOfApplication() {
+        return applicationStatus.getApplicationStatus();
```

Comment:
Perhaps consider renaming `ApplicationStatus#getApplicationStatus()` to `ApplicationStatus#getStatusOfApplication()` so as to disambiguate is from`Applicant#getApplicationStatus()`.

Reply:
Noted, changes made